widgets/environmentButton/src/widget.tsx

```diff
@@ -26,11 +32,11 @@ Stage.defineWidget<never, EnvironmentButtonWidget.Data, ButtonConfiguration>({
         const searchActions = new SearchActions(toolbox);
 
         return searchActions.doListBlueprints({
-            filterId: environmentFilterId,
             params: {
                 _include: 'id',
                 _size: 1
-            }
+            },
+            filterRules: [environmentFilterRules]
```

Comment:
I realized that we have something similar in `app/widgets/common/filters/common.ts`, so maybe:
```suggestion
            filterRules: [Stage.Common.Filters.environmentFilterRule]
```
will work here?

Reply:
Oh, that's a very handy suggestion! 💪 
Just checked and it's working as expected 🎉

Reply:
Was introduced in 607c717222ee69711aeb7f1116c19618d041de97